Formula/m/mame.rb

```diff
@@ -67,8 +68,8 @@ def install
                    "USE_LIBSDL=1",
                    "USE_SYSTEM_LIB_EXPAT=1",
                    "USE_SYSTEM_LIB_ZLIB=1",
-                   "USE_SYSTEM_LIB_ASIO=1",
                    "USE_SYSTEM_LIB_LUA=",
```

Comment:
Unrelated, but I'm puzzled by this: `USE_SYSTEM_LIB_LUA=`
Does it mean 1 (on) or 0 (off)?